lib/components/MenuOptions.js

```diff
@@ -95,8 +95,6 @@ var MenuOptions = module.exports = React.createClass({
         id={this.context.id}
         role="menu"
         tabIndex="-1"
-        aria-expanded={this.context.active}
```

Comment:
yeah, figured it was more consistent to have it on an element that will always be rendered :)